pkg/controller/plan/migration.go

```diff
@@ -1501,7 +1501,7 @@ func (r *Migration) updateCopyProgress(vm *plan.VMStatus, step *plan.Step) (err
 			}
 			conditions := dv.Conditions()
 			switch dv.Status.Phase {
-			case cdi.Succeeded, cdi.Paused:
+			case cdi.Succeeded, cdi.Paused, cdi.PendingPopulation:
```

Comment:
this should hold only when migrating from vSphere, is it really limited to vSphere this way?